webapp/app/components/Message/Message.component.js

```diff
@@ -0,0 +1,55 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Map } from 'immutable';
+import moment from 'moment';
+import { ListItem, ListItemText } from 'material-ui/List';
+
+import { Author, Data, MessageContent } from './Message.styles';
+import { UserAvatar } from '../UserAvatar/UserAvatar.component';
+
+
+export class Message extends PureComponent {
+  static propTypes = {
+    message: PropTypes.instanceOf(Map),
+  };
+
+  get time() {
+    const data = moment(this.props.message.get('publicationTime'));
+
+    if (moment().isSame(data, 'd')) {
+      return moment(data, 'hmm').format('HH:mm');
+    }
+    return moment(data, 'hmm').format('HH:mm DD.MM.YYYY');
+  }
+
+  get author() { return this.props.message.get('author'); }
```

Comment:
czy te gettery sa potrzebne skoro korzystasz z tych danych tylko w jednym miejscu?

Reply:
getterow raczej uzywalbym do pobierania jakis propsow danego komponentu, do zwracania komponentow zazwyczaj uzywamy funkcji z render w nazwie

Reply:
Wychodząc trochę naprzeciw powyciągałem to pobieranie props'ów do getterów, zakładam, że na późniejszym etapie projektu (pewnie przy następnej PR) to pobieranie autora będzie już bardziej złożone. Prawdopodobnie coś w stylu:
`this.props.users.getIn([this.props.message.get('userID'), 'displayName']);`
Co o tym myślisz?